frontend/src/metabase/dashboard/containers/CreateDashboardForm.tsx

```diff
@@ -93,9 +72,10 @@ function CreateDashboardForm({
 
   const handleCreate = useCallback(
     async (values: CreateDashboardProperties) => {
-      const action = await handleCreateDashboard(values);
-      const dashboard = Dashboards.HACK_getObjectFromAction(action);
```

Comment:
(I just love how this function has a "// for now just use this function until we figure out a cleaner way to do this" comment from 7 years ago 🤣 - good cleanup!)